GroceryMarket/Animation.swift

```diff
@@ -17,17 +17,36 @@ class Animation : NSObject {
 
 @objcMembers
 class TabBarFormat : NSObject {
-//    override func viewDidLoad(){
-//        super.viewDidLoad()
-//
-//        self.ChangeRadiusOfTabBarItem()
-//    }
     
     func ChangeRadiusOfTabBarItem(_ tabBar: UITabBar)  {
         tabBar.layer.masksToBounds = true;
         tabBar.isTranslucent = true;
-        tabBar.layer.cornerRadius = 30
+        tabBar.layer.cornerRadius = 50
         tabBar.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
         
     }
+    
+    func ChangeUnselectedColor(_ tabBar: UITabBar){
+        tabBar.unselectedItemTintColor = #colorLiteral(red: 0.5843137503, green: 0.8235294223, blue: 0.4196078479, alpha: 1)
+    }
+    
+    func ChangeHeightOfTabbar(_ tabBar: UITabBar){
```

Comment:
You can make the three functions `changeRadiusOfTabBarItem`, `changeUnselectedColor` and `changeHeightOfTabBar` private functions, and then create a public function `setupTabBar` which calls these three functions. This case, you reduce the number of public functions as much as possible.